src/postman/operation.ts

```diff
@@ -57,6 +82,15 @@ function findItem(collection: Collection, method: string, path: string): Item |
   return found;
 }
 
+// this func ensures that both this securityScheme and the one in HttpService share exactly the sme keys
+function findPostmanSecurityScheme(auth: RequestAuth, collection: Collection) {
+  const securityScheme = transformSecurityScheme(auth, () => '1');
+  if (!securityScheme) return;
+
+  const allSecuritySchemes = transformSecuritySchemes(collection);
```

Comment:
Hmm, computing this every time we need to find a security scheme seems at glance highly inefficient.